1.illumination_correction/README.md

```diff
@@ -0,0 +1,77 @@
+# Cell Profiler Illumination Correction Pipeline
+In this module, I present the 10 CellProfiler pipelines I created using various manual parameters to create the illumination function and correct the raw images. The chosen image for comparison is **D6_01_1_4_DAPI_001.tif** due to the large artifact present in the bottom right corner.
+
+Within the [Images folder](1.illumination_correction/Images), there is only the one image from the data set that I chose to compare with each other. 
+
+## Opinions for each pipeline
+
+Below I provide my opinions regarding each pipeline, how the illumination correction function worked on the chosen image, and how it worked in general with all of the images.
+
+### Regular -> Each image -> Median (Manual 40) -> Divide
+    - Fast runtime.
+    - All images are very dim, hard to tell if illumination correction occurred.
+    - Must brighten images to see 
+    - For the chosen image, once brightened in ImageJ, it looks like the nuclei have become darker in the center and then brighter on the edges.
+        - Due to this, this pipeline is likely not viable as it could impact the biological anaylsis.
+
+### Regular -> All: Across Cycles -> Median (Manual 40) -> Divide 
+
+    - Much slower runtime, approximately 12 minutes.
+    - Creates a weird "shadow" effect, where in all the images, there looks to be dark shadows of the object being stained for based on the channel.
+    - This shadow issue can really be seen in the chosen image, where the shadows can be seen all over the artifact.
+    - Seems to be that the "All: Across Cycles" parameter causes there to be reminants of other images on each image
+
+### Background (Block 60)-> Each image -> Median (Manual 40) -> Divide
+
+    - Fast runtime.
+    - Creates faint boxes over objects for majority of the images in the data set.
+    - For the chosen image, it is obvious the huge squiggly lines over the artifact that expand over the nuclei in the top right corner.
+    - Note: Divide parameter is not what is recommended for using the Background parameter, it should be Subtract 
```

Comment:
the divide parameter is not what "we" recommend? Or is it not what the CellProfiler manual recommends?